pkg/jsonpath/v1alpha1/json_path.go

```diff
@@ -20,10 +20,11 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	. "github.com/openebs/maya/pkg/msg/v1alpha1"
+	"reflect"
+
```

Comment:
Please remove this extra blank line.

Reply:
this line was generated by the linter. the third party packages should be the last ones on imports list.